frame/contracts/src/wasm/prepare.rs

```diff
@@ -1021,7 +791,7 @@ mod tests {
 				(func (export "deploy"))
 			)
 			"#,
-			Err("Validation of new code failed!")
+			Err("Can't load the module to wasmi!")
```

Comment:
Same for the other occurences

```suggestion
			Err("Can't load the module into wasmi!")
```